src/main/java/emu/grasscutter/server/packet/send/PacketPlayerWorldSceneInfoListNotify.java

```diff
@@ -21,6 +23,9 @@ public PacketPlayerWorldSceneInfoListNotify(Player player) {
             if (scene.getSceneType() != SceneType.SCENE_WORLD) continue;
 
             var worldInfoBuilder = new PlayerWorldSceneInfo();
+            var isSceneLocked = player.getUnlockedScenes()
```

Comment:
This should probably be the host, not the player, since in official coop you can enter scenes the other person unlocked, that you haven't unlocked yet.

Reply:
Did this. I made sure not to use computeIfAbsent if the player isn't the host. The player should not be modifying the host's unlockedScenes with the player's config.